docs/src/components/GestureExamples/FlingExample/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { StyleSheet } from 'react-native';
+import stylesWeb from './styles.module.css';
+import {
+  GestureHandlerRootView,
+  GestureDetector,
+  Gesture,
+} from 'react-native-gesture-handler';
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+import Hand from '@site/static/img/hand-one.svg';
+import Arrows from '@site/static/img/two-arrows.svg';
+import { Directions } from 'react-native-gesture-handler';
+import { RADIUS, isInsideCircle, useStylesForExample } from '../utils';
+
+export default function FlingExample() {
+  const colorModeStyles = useStylesForExample();
+  const [showHand, setShowHand] = useState(true);
+  const startPositionX = useSharedValue(0);
+  const endPositionX = useSharedValue(0);
+  const startPositionY = useSharedValue(0);
+  const endPositionY = useSharedValue(0);
+  const positionX = useSharedValue(0);
+  const positionY = useSharedValue(0);
+
+  const config = { duration: 100 };
+
+  const flingGesture = Gesture.Fling()
+    .direction(
+      Directions.UP | Directions.DOWN | Directions.LEFT | Directions.RIGHT
+    )
+    .onBegin((e) => {
+      setShowHand(false);
+      startPositionX.value = e.x;
+      startPositionY.value = e.y;
+    })
+    .onEnd((e) => {
+      endPositionX.value = e.x;
+      endPositionY.value = e.y;
+    })
+    .onStart((e) => {
+      if (endPositionX.value === 0) endPositionX.value = e.x;
+      if (endPositionY.value === 0) endPositionY.value = e.y;
```

Comment:
```suggestion
    .onStart((e) => {
      endPositionX.value = e.x;
      endPositionY.value = e.y;
```
please make this change to make it work a little bit more smooth